Gemfile.lock

```diff
@@ -11,6 +11,7 @@ GEM
     ast (2.4.2)
     attr_extras (6.2.5)
     bson (4.2.2)
+    bson (4.2.2-java)
```

Comment:
I think we shouldn't be pulling in the `*-java` gems

Reply:
I think this just points out that if you're bundling using jruby, this is the platform-dependent gem you'll end up with. Similarly with Windows ruby and `nokogiri (1.13.4-x86-mingw32)`. I could be wrong.

Reply:
I ran `rbenv version` from the root of `connectors` and I got

```
2.6.9 (set by /Users/chenhui.wang/src/github.com/elastic/connectors/.ruby-version)
```

I think I'm not using jruby here?

Reply:
if you restore the `Gemfile.lock` to how it is in `main`, then re-run `make install` from your root, do you still get the `-java` changes? I've had this happen when my connectors checkout is correct, but I run `make install` or `bundle install` from inside my IDE terminal.

Reply:
Yes, I still got thoses `-java` changes.